dependencies/pom.xml

```diff
@@ -1101,6 +1104,22 @@
                 <artifactId>neo4j-java-driver</artifactId>
                 <version>${version.lib.neo4j}</version>
             </dependency>
+            <!-- Bean Validation -->
+            <dependency>
+                <groupId>javax.validation</groupId>
+                <artifactId>validation-api</artifactId>
+                <version>${version.lib.validation-api}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.hibernate.validator</groupId>
+                <artifactId>hibernate-validator</artifactId>
+                <version>${version.lib.hibernate-validator}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.glassfish</groupId>
+                <artifactId>javax.el</artifactId>
+                <version>${version.lib.el}</version>
+            </dependency>
```

Comment:
Correct me please if I am wrong, but should not this be based on Jakarta API?

Reply:
You are completely right! My bad. Fixed!